Source/DataSources/GeoJsonDataSource.js

```diff
@@ -627,7 +626,6 @@ define([
             },
             set : function(value) {
                 defaultFill = value;
-                defaultFillMaterialProperty = new ColorMaterialProperty(defaultFill);
```

Comment:
It's weird that this got picked up, but `defaultStrokeMaterialProperty` and `defaultStrokeWidthProperty` variables in this file (which are also unused) didn't.  Any idea why?  Can you remove them as well.  Thanks.